breez/breez.proto

```diff
@@ -121,13 +124,27 @@ message LSPInformation {
   /// The minimum value in millisatoshi we will require for incoming HTLCs on the channel.
   int64 min_htlc_msat = 10 [json_name = "min_htlc_msat"];
 
-  int64 channel_fee_permyriad = 11;
+  int64 channel_fee_permyriad = 11 [deprecated = true];
 
   bytes lsp_pubkey = 12;
   // The channel can be closed if not used this duration in seconds.
-  int64 max_inactive_duration = 13;
+  int64 max_inactive_duration = 13 [deprecated = true];
 
-  int64 channel_minimum_fee_msat = 14;
+  int64 channel_minimum_fee_msat = 14 [deprecated = true];
+
+  repeated OpeningFeeParams opening_fee_params_menu = 15;
```

Comment:
The mobile app only needs the short validity. MIght be a good idea to only expose the open channel fee for short validity here. Otherwise we will need to duplicate the logic of choosing the short validity in the flutter side as well.
What do you think @JssDWt ?